### PR TITLE
[DF-2935] - O365 Virus Total

### PR DESCRIPTION
**Pull request by jmcadams-r7**
## Description: 

This PR adds an O365 Email Analysis workflow that uses Virus Total for URL and File analysis. 

#### Workflow Builder

workflow1
workflow2
workflow3

url_analysis_outter_loop
url_analysis_inner_loop

#### Artifact

artifacts1
artifacts2
artifacts3

### CODE REVIEW

File: workflows/Office_365_Analysis_with_Virus_Total/Office 365 Analysis Virus Total.icon [hunk 1]
```diff
@@ -0,0 +1,3349 @@
...
+          "3747b60f-4236-4393-a4da-bc760bbf86d6": {
+            "nodeId": "3747b60f-4236-4393-a4da-bc760bbf86d6",
+            "name": "VT Look for Hash",
+            "type": "action",
+            "plugin": {
+              "name": "VirusTotal",
+              "slugVendor": "rapid7",
+              "slugName": "virustotal",
+              "slugVersion": "5.0.0",
+              "imageData": "https://us.cdn-assets.connect.ins
```
Comment (mrinehart-r7) [inline workflows/Office_365_Analysis_with_Virus_Total/Office 365 Analysis Virus Total.icon]:
```suggestion
            "name": "Is URL Malicious",
```
Want a ? at the end of that title? I see that in some of your steps.

Comment (mrinehart-r7) [reply workflows/Office_365_Analysis_with_Virus_Total/Office 365 Analysis Virus Total.icon]: Also, don't trust this code suggestion. GitHub is borked with the # of lines here.

Comment (mrinehart-r7) [inline workflows/Office_365_Analysis_with_Virus_Total/Office 365 Analysis Virus Total.icon]:
```suggestion
            "name": "Was Overall Mail Malicious",
```

File: workflows/Office_365_Analysis_with_Virus_Total/Office 365 Analysis Virus Total.icon [hunk 2]
```diff
@@ -0,0 +1,3349 @@
...
+  "kom": {
+    "komandVersion": "1.38.0-0-g8d14f",
+    "komFileVersion": "2.0.0",
+    "exportedAt": "2019-09-30T18:53:19.746094221Z",
+    "workflowVersions": [
+      {
+        "name": "Office 365 Analysis Virus Total",
+        "type": "runnable",
+        "version": "",
+        "description": "This workflow is designed to automatically analyze incoming emails to determine if they are malicious. This workflow assumes a user is using their report phishing button to send an email to an administrative account with the suspicious email attached. The workflow will then analyze any URLs or files in the attached email to find malicious links. If any malicious indicators are found the email will be remediated across the organization.",
```
Comment (jdf400-rapid7) [inline workflows/Office_365_Analysis_with_Virus_Total/Office 365 Analysis Virus Total.icon]: What is the remediation?

Comment (jmcadams-r7) [reply workflows/Office_365_Analysis_with_Virus_Total/Office 365 Analysis Virus Total.icon]: Block Sender / Compliance Search, Purge.

**Comment by lmilby-r7** [comment]
What's the criteria for the malicious URL for file? Should have be a human decision?

**Comment by jmcadams-r7** [comment]
# Update Malicious File Report Artifact

artifacts2

**Pull request merged**